Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -93,6 +93,44 @@ public async Task it_remembers_state_between_submissions(Language language)
                 .Be(5);
         }
 
+        // Option 1: inline switch
+        [Theory]
+        [InlineData(Language.CSharp)]
+        //[InlineData(Language.FSharp)]    Todo: Issue 695 - dotnet-interactive with an F# notebook does not load System.Text.Json #695
+        public async Task it_can_reference_system_text_json(Language language)
+        {
+            var source = language switch
+            {
+                Language.FSharp => new[]
+                {
+@"open System.Text.Json;
+let jsonException = JsonException()
+let location = jsonException.GetType().Assembly.Location
+location.EndsWith(""System.Text.Json.dll"")"
+                },
+
+                Language.CSharp => new[]
+                {
+@"using System.Text.Json;
+var jsonException = new JsonException();
+var location = jsonException.GetType().Assembly.Location;
+location.EndsWith(""System.Text.Json.dll"")"
+        }
+            };
+
+            var kernel = CreateKernel(language);
+
+            await SubmitCode(kernel, source);
+
+            KernelEvents
```

Comment:
You can also do this, which will be more informative if the event is missing:

```csharp
KernelEvents
  .Should()
  .ContainSingle<ReturnValueProduced>()
  .Which
  .Value
  .Should()
  .BeTrue();
```

Reply:
I like it ...